.github/workflows/main.yml

```diff
@@ -8,3 +8,14 @@ jobs:
             - uses: actions/checkout@master
             - name: PHPStan
               uses: docker://oskarstark/phpstan-ga
+              with:
+                  args: analyze
+    php-cs-fixer:
+        name: PHP-CS-Fixer
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@master
+            - name: PHP-CS-Fixer
+              uses: docker://oskarstark/php-cs-fixer-ga
+              with:
+                args: --config=.project.php_cs --diff --dry-run
```

Comment:
`project.php_cs` should probably be `.php_cs` ? :slightly_smiling_face:

Reply:
Yeah, I just copied code. =)